test/sshkit/context_test.exs

```diff
@@ -42,5 +42,23 @@ defmodule SSHKit.ContextTest do
       assert command =~ ~r/export CI="true"/
       refute command =~ ~r/NODE_ENV/
     end
+
+    test "build with nil env" do
+      command =
+        @empty
+        |> SSHKit.env(nil)
+        |> Context.build("ls")
+
+      assert command == "/usr/bin/env ls"
+    end
+
+    test "build with empty env" do
+      command =
+        @empty
+        |> SSHKit.env(%{})
+        |> Context.build("ls")
+
+      assert command == "/usr/bin/env ls"
+    end
```

Comment:
I see we're using `SSHKit` in one of the tests above as well ("build with overwritten env"). I think we should avoid this when unit-testing `Context` and instead that test-case should go into `sshkit_test.exs`.

Does that make sense? If so, would you mind updating your PR to instead use the `%Context{@empty | env: …}` approach from the other tests?

Reply:
Awesome! I'll can take care of moving the old test later on: https://github.com/bitcrowd/sshkit.ex/issues/103